content/stability-working-group/index.md

```diff
@@ -0,0 +1,73 @@
+
+By ecosystem we mean the broadest most inclusive set possible. This includes compiler and tooling implementors, industrial users across the spectrums of experience and enthusiasm, language designers, library authors and maintainers, researchers, students, anyone else working with the language, and anyone who works with the aforementioned. The needs of those groups are quite diverse and can often be in tension. So one of the first things to keep in mind for stability is how to balance those needs. It is simply required to be empathetic to all of those groups.
+
+Recognizing this diversity means realizing where there are gaps in representation in the ecosystem today. Further we must remain cognizant that many of those groups simply will not increase their engagement with the rest of the ecosystem. How then can we possibly be aware of issues or concerns if those impacted are not engaged? There must be a bridge to them. Anyone involved in being that bridge then must navigate conversations to represent needs that might be, irrelevant to them or even in tension with their own.
+
+Now with the ecosystem in mind, what exactly is stability?
+
+For many they will read that word and think immutability. We are talking about the Haskell ecosystem after all. However, stability here does not mean a complete lack of change. To forego change entirely would mean to not be able to incorporate new ideas or react to the rest of the world changing. If this was the case Haskell would never have grown to include a huge number of things, including the IO monad! Interaction with he rest of the world will continue to require changes, for example the first version of The Unicode Standard was published after Haskell 1.0!
+
+So we accept that not only will change happen, it must. This leads to how I would describe stability: the ability to react to change in an informed and graceful way. With change as a given, we must be able to react to it. But the other parts are also extremely important. Change can be extremely disruptive. In the ecosystem, change might mean:
+ - industrial users need to spend time away from their employer's goals
+ - library and tooling maintainers have to adapt and support multiple paths, taking away from feature development
+ - academic materials might need to be rewritten
+ - any other number of issues that are difficult to predict
+
+But we just said that change is going to happen! As such, _ecosystem stability_ is managing the *impact* of change as felt by all members described above . In order to reduce the negative impacts of change to the ecosystem there must be a graceful handling path, not a lack of change entirely. For example, deprecating something before removal. Another important factor here is the timing. Being predictable in change management means that the various groups can schedule work rather than being interrupted. This is why it is critical to be able to handle change as gracefully as possible.
```

Comment:
```suggestion
But we just said that change is going to happen! As such, _ecosystem stability_ is managing the *impact* of change as felt by all members described above. In order to reduce the negative impacts of change to the ecosystem there must be a graceful handling path, not a lack of change entirely. For example, deprecating something before removal. Another important factor here is the timing. Being predictable in change management means that the various groups can schedule work rather than being interrupted. This is why it is critical to be able to handle change as gracefully as possible.
```